### PR TITLE
Portfolio Pull Request

### PR DESCRIPTION
**Pull request by mar4444**
What I did to my project:

- Created index.html file.
- Created style.css file.
- Created script.js file which I used to create hamburger menu.
- GitHub Flow.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.menu-sec').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```
Comment (usorfaitheloho) [inline script.js:12]: You did well with your mobile popup, kindly implement the following changes so as to enable you have a better design and user experience 😄 :

- According to the requirement, when a user clicks any of the mobile menu options , it should direct you to that part of the page. For example, when I click on the `About` it takes me to the about section, but when I click on `Contact` it doesn't take me to the contact section and also, the `Portfolio` should take me to the project section 

Please kindly note that the background for the popup menu is blur in the Figma design given, kindly implement this for your project 😄 

Figma design :
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mar4444  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,812 @@
...
+    left: -600px;
+    top: 0;
+    gap: 0;
+    background-color: #6070ff;
+    opacity: 0.9;
+    height: 100%;
+    width: 100%;
+    transition: 0.3s;
+  }
+
```
Comment (Munsa1) [inline style.css:54]: - [ ] (optional) Good job making your mobile menu design transparent  so as to match the given design. To match the Figma design even more you could create an empty `div` inside the header tag and give it a class name of overlay. Then you counld give it the following CSS properties so as to fully achieve the blurred background. You could then make the class show when the menu is clicked. You should try it.


CSS
```
.overlay {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: #your color;
  backdrop-filter: blur(3px);
  z-index: 5;
}
```

HTML

```
      <header>
       <div class="overlay hidden"></div>
      </header>
```

**Review by Munsa1 — APPROVED** [review]
Hi @mar4444, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes By Previous Reviewer :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**